src/SharedComponents/AstroVirtualAssistant/CommandMessageProcessor.tsx

```diff
@@ -54,15 +54,19 @@ export const commandMessageProcessor: MessageProcessor = async (message, options
         await manageOrg2FaCommandProcessor(message.command, options);
         break;
       case CommandType.CREATE_SERVICE_ACCOUNT: {
-        const serviceAccInfo = await createServiceAccProcessor(message.command, options);
-        if (serviceAccInfo.status < 300) {
-          options.addBanner('create_service_account', [
-            serviceAccInfo.data.name,
-            serviceAccInfo.data.description,
-            serviceAccInfo.data.clientId,
-            serviceAccInfo.data.secret,
-          ]);
-        } else {
+        try {
+          const serviceAccInfo = await createServiceAccProcessor(message.command, options);
+          if (serviceAccInfo.status < 300) {
+            options.addBanner('create_service_account', [
+              serviceAccInfo.data.name,
```

Comment:
Was the error you were getting with parsing the service account info?

Reply:
It just straight up saying forbidden 403 and is saying Uncaught in Promise. User probably needs some special permission, not sure.